### PR TITLE
main Landing Page Complete

### PR DESCRIPTION
**Pull request by MayurChawla**
This is the landing Page. Please review

landing page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+}
+.about-UI .ad-image {
+  width: 33rem;
+}
+.about-UI .ad-image img {
+  width: 33rem;
+  height: auto;
+}
+.footer {
+  color: white;
```
Comment (aurobindoGupta) [inline styles.css:125]: use variable

**Review by aurobindoGupta — APPROVED** [review]
looks good

File: styles.css (hunk 1 above)
Comment (MayurChawla) [reply styles.css:125]: yes

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+}
+::-webkit-scrollbar {
+  width: 0px;
+}
+html {
+  width: 100%;
+  height: 100%;
+}
+body {
+  display: flex;
```
Comment (nareshpoonia) [inline styles.css:32]: You can consider using flex-col utility class instead, since you'll be using it many times

**Review by nareshpoonia — COMMENTED** [review]
Code looks clean, just one comment of using utility classes.
Good to merge

File: styles.css (hunk 2 above)
Comment (MayurChawla) [reply styles.css:32]: yes, good idea. Thanks

**Pull request merged**